packages/webdriverio/src/commands/element/getLocation.ts

```diff
@@ -27,16 +36,11 @@ import { getElementRect } from '../../utils'
  * @uses protocol/elementIdLocation
  * @type property
  */
-export default async function getLocation (
+export async function getLocation<T extends keyof Coordinates> (
```

Comment:
why export twice this method twice?

Reply:
Hi! By twice, were you referring to the default export below or the named exports above?